File: app/src/main/java/com/bubelov/coins/api/rates/CoinbaseApi.java
package com.bubelov.coins.api.rates;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Author: Igor Bubelov
 * Date: 9/28/15 9:26 PM
 */

public interface CoinbaseApi {
    @GET("products/BTC-USD/ticker")
    Call<CoinbaseTicker> getTicker();
}

File: app/src/main/java/com/bubelov/coins/api/rates/CoinbaseApi.kt
package com.bubelov.coins.api.rates

import retrofit2.Call
import retrofit2.http.GET

/**
 * @author Igor Bubelov
 */

interface CoinbaseApi {
    @GET("exchange-rates?currency=BTC")
    fun getExchangeRates(): Call<Response>
}

data class Response (val data: Data)

data class Data (val currency: String, val rates: Map<String, Double>)